binscripts/rvm-installer

```diff
@@ -219,7 +218,7 @@ __rvm_curl()
 )
 
 rvm_error()  { printf "ERROR: %b\n" "$*"; }
-__rvm_which(){   which "$@" || return $?; true; }
+__rvm_which(){ command -v "$@" 2>/dev/null; }
```

Comment:
If we're guaranteed bash (not just sh), we can use `type -P` to get the path to an external command. As for `builtin`, it's just as easy to goof around with that as `command`, right?

```builtin() { eval "$1() { echo lol; }"; }```